frontend/src/components/editCard/VoteBar.tsx

```diff
@@ -26,13 +26,23 @@ const VoteBar: FC<Props> = ({ edit }) => {
   const [vote, setVote] = useState<VoteTypeEnum | null>(userVote?.vote ?? null);
   const [submitVote, { loading: savingVote }] = useVote();
 
-  if (
-    edit.status !== VoteStatusEnum.PENDING ||
-    !canVote(auth.user) ||
-    auth.user?.id === edit.user?.id
-  )
+  if (edit.status !== VoteStatusEnum.PENDING || !canVote(auth.user))
     return <></>;
 
+  if (auth.user?.id === edit.user?.id)
+    return (
+      <div className={CLASSNAME}>
```

Comment:
I think it looked bad without the buttons when I used the same classes here. One sec...

Reply:
![image](https://user-images.githubusercontent.com/25374869/146639805-9eac06f9-0b84-4f3c-8555-2f934181520e.png)
Looks ok on my end.

Reply:
Before
![before](https://user-images.githubusercontent.com/66393006/146640164-765e4be2-5343-40bd-a614-7c6820d137c8.png)
After
![after](https://user-images.githubusercontent.com/66393006/146640166-3cc949a5-5d7a-461f-bbcc-78eacca17d16.png)

Reply:
Yup, that's better. 👍